gregBorchers/hwWeek1.js

```diff
@@ -0,0 +1,215 @@
+
+// ### Homework #1
+// (Due Monday 1/12)
+
+// You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+// For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+// Please also read the [learning objectives](objectives.md) for this week.
+
+// ---
+
+// **1)** (_Difficulty: mixed_)
+// Predict the output of each of the following expressions, and explain the rules which determine it.
+// If the output depends on the value of x, identify the conditions when the expression will output true.  Assume the cases are independent, and x is reset to an unknown value before each.
+
+// Some of these are tricky!  Don't trust your first instinct.  
+
+
+// **a)** `"1" == 1`
+true
+
+// **b)** `"1" === 1`
+false
+
+// **c)** `x == 'x'`
+true // only when x is assinged to 'x' beforehand like x = 'x'
+
+// **d)** `x == (x+'')`
+true // no matter what is assigned to x, adding '' to is is like adding nothing, so x remains the same (no side effect)
```

Comment:
There are a couple of edge cases here: `NaN` is never equal to anything, including itself, and booleans will autoconvert to strings when added to strings.